### PR TITLE
Prepare for the 2.3.1 release

### PR DESCRIPTION
**Pull request by daschuer**

### CODE REVIEW

File: content/news/xxxx-xx-xx-mixxx-2-3-1-released.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+tags: 2.3, 2.3.1, release announcement
+status: draft
+comments: yes
+
+We are happy to announce first minor release of [Mixxx 2.3.1](https://github.com/mixxxdj/mixxx/releases/tag/2.3.1)!
+You will find the download links [here]({filename}/pages/download.md#stable) on Windows, macOS and Linux.
+
+This release comes with new and updated controller mappings, fixes packaging issues and includes some bugfixes.
+
+Mixxx now includes mappings for the [Numark DJ2GO2 Touch](https://manual.mixxx.org/2.3/en/hardware/controllers/numark_dj2go2_touch.html) controller and the [Numark Mixtrack Pro FX](https://manual.mixxx.org/2.3/en/hardware/controllers/numark_mixtrack_pro_fx.html) controller. The [Behringer DDM4000](https://manual.mixxx.org/2.3/en/hardware/controllers/behringer_ddm4000.html) and the [Traktor S3](https://manual.mixxx.org/2.3/en/hardware/controllers/native_instruments_traktor_kontrol_s3.html) mapping has been improved with Mixxx 2.3.
```
Comment (ronso0) [inline content/news/xxxx-xx-xx-mixxx-2-3-1-released.md]:
```suggestion
Mixxx now includes mappings for the [Numark DJ2GO2 Touch](https://manual.mixxx.org/2.3/en/hardware/controllers/numark_dj2go2_touch.html) controller and the [Numark Mixtrack Pro FX](https://manual.mixxx.org/2.3/en/hardware/controllers/numark_mixtrack_pro_fx.html) controller. The [Behringer DDM4000](https://manual.mixxx.org/2.3/en/hardware/controllers/behringer_ddm4000.html) and the [Traktor S3](https://manual.mixxx.org/2.3/en/hardware/controllers/native_instruments_traktor_kontrol_s3.html) mapping have been improved with Mixxx 2.3.
```

**Comment by ronso0** [comment]
Thank you!
Shall we mention some important bugfixes, too?
https://launchpad.net/mixxx/+milestone/2.3.1

I'm also removing open bugs from the 2.3.1 milestone

**Comment by daschuer** [comment]
> Shall we mention some important bugfixes, too?

I have thought about it. What is really important are the issues around the AutoDJ, unfortunately the most important fix https://github.com/mixxxdj/mixxx/pull/4319 was merged after splitting out 2.3.1.
To make this a success story, we can announce it with 2.3.2. or release the current 2.3.1 branch instead. 

I would prefer it anyway. This will also release
#4287 Numark_DJ2GO2_Touch (a fix for the just announced mapping)
#4324 #4021 Denon_MC7000_mapping
#4312 daschuer/lp1943320
#4296 ninomp/fix-playerinfo
#4246 michalvankodev/hercules-inpulse300-mappings
#4288 ninomp/fix-beatgrid-reanalyze-imported

What do you think?

**Comment by daschuer** [comment]
This is the PR for the 2.3.2 changes https://github.com/mixxxdj/mixxx/pull/4333

**Comment by daschuer** [comment]
This was merged and sqashed to https://github.com/mixxxdj/website/pull/257

**Pull request closed without merging**